lib/pod/command/check.rb

```diff
@@ -79,11 +80,52 @@ def find_differences(config, development_pods)
         end.compact
       end
 
-      def development_result(spec_name, external_source)
-        if @check_command_verbose
-          "#{spec_name} #{external_source}"
+      def get_files_newer_than_lockfile_for_podspec(config, development_pod)
+        files_for_podspec = get_files_for_podspec(development_pod[:path])
+
+        lockfile_mtime = File.mtime(config.lockfile.defined_in_file)
+        podspec_file = get_podspec_for_file_or_path(development_pod[:path])
+        podspec_dir = Pathname.new(File.dirname(podspec_file))
+        files_for_podspec
+            .select {|f| File.mtime(f) >= lockfile_mtime}
+            .map {|f| Pathname.new(f).relative_path_from(podspec_dir).to_s}
+      end
+
+      # Returns an array of all files pointed to by the podspec
+      def get_files_for_podspec(podspec_file)
+        podspec_file = get_podspec_for_file_or_path(podspec_file)
+
+        development_pod_dir = File.dirname(podspec_file)
+        spec = Specification.from_file(podspec_file)
+
+        # Gather all the dependencies used by the spec, across all platforms, and including subspecs.
+        all_files = [spec, spec.subspecs].flatten.map { |a_spec|
+          a_spec.available_platforms.map { |platform|
+            accessor = Sandbox::FileAccessor.new(Sandbox::PathList.new(Pathname.new(development_pod_dir)), a_spec.consumer(platform))
+            [
+                accessor.vendored_frameworks,
```

Comment:
This makes me a little nervous because if new things are added in the future, they won't be picked up here. Is there anyway to get 'all files'?